lib/recaptcha/client_helper.rb

```diff
@@ -25,9 +25,9 @@ def recaptcha_tags(options = {})
                 <div style="width: 302px; height: 422px; position: absolute;">
                   <iframe
                     src="#{fallback_uri}"
-                    frameborder="0" scrolling="no"
-                    style="width: 302px; height:422px; border-style: none;">
-                    title="ReCAPTCHA"
+                    scrolling="no"
+                    style="width: 302px; height: 422px; border-style: none; border: 0;">
+                    ReCAPTCHA
```

Comment:
that has to stay the title ?

Reply:
I moved the title to the name attribute. It's ok?